setup.py

```diff
@@ -2,19 +2,18 @@
 
 setup(name = "cf-view",
     long_description = "GUI for viewing CF files",
-    version      = "3.2.25",
+    version      = "3.3.0",
     description  = "GUI for viewing CF files",
     author       = "Andy Heaps",
-    maintainer   = "Andy Heaps",
-    maintainer_email  = "andy.heaps@ncas.ac.uk",
-    author_email = "andy.heaps@ncas.ac.uk",
+    maintainer   = "Sadie Bartholomew",
+    maintainer_email  = "sadie.bartholomew@ncas.ac.uk",
     url          = "http://ajheaps.github.io/cf-view",
-    download_url = "https://github.com/ajheaps/CF-View",
+    download_url = "https://github.com/ncas-cms/cf-view",
```

Comment:
```suggestion
    download_url = "https://github.com/NCAS-CMS/cf-view",
```

(GitHub domain for organisation and repo etc. is case sensitive)